styles/style.css

```diff
@@ -0,0 +1,173 @@
+html {
+    height: 100%;
+}
+
+body {
+    height: 100%;
+}
+
+.body {
+    display: grid;
+    grid-template-columns: 200px 1fr;
+    grid-template-areas:
+        'user-name'
+        'greeting'
+        'top-mixes'
+        'made-for'
+        'recents'
+        'footer';
+}
+
+main #sidebar {
+    font-family: sans-serif;
+    background: black;
+    height: 730px;
+    position: static;
+}
+
+#logo img {
+    height: 40px;
+    object-fit: contain;
+}
+
+.sidebar-links {
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr;
+    color: lightslategrey;
+    padding: 5px;
+}
+
+.sidebar-links:hover {
+    color: white;
+}
+
+#install-app {
+    color: lightslategray;
+    position: relative;
+    top: 400px;
+}
+
+#install-app:hover {
+    color: white;
+}
+#user-name {
+    font-weight: bold;
+    font-family: sans-serif;
+    background: rgb(216, 66, 66);
+    text-align: right;
+    height: 40px;
+}
+
+#greeting {
+    font-weight: bold;
+    font-family: sans-serif;
+    background: burlywood;
+    height: 160px;
+    margin: 10px;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: 25% 25% 25% 25%;
+    gap: 5px 10px;
+    grid-area: greeting;
+    grid-template-areas:
+        'greeting greeting'
+        'item1 item4'
+        'item2 item5'
+        'item3 item6';
+}
+
+#top-mixes {
+    font-weight: bold;
+    font-family: sans-serif;
+    background: lightblue;
+    height: 160px;
+    margin: 10px;
+    display: grid;
+    grid-template-columns: 25% 25% 25% 25%;
+    grid-template-rows: 33.3% 33.3% 33.3%;
+    grid-area: top-mixes;
+    grid-template-areas:
+        'top-mixes top-mixes top-mixes top-mixes'
+        'item1 item2 item3 item4';
+}
+
+#made-for {
+    font-weight: bold;
+    font-family: sans-serif;
+    background: violet;
+    height: 160px;
+    margin: 10px;
+
+    display: grid;
+    grid-template-columns: 25% 25% 25% 25%;
+    grid-template-rows: 33.3% 33.3% 33.3%;
+    grid-area: made-for;
+    grid-template-areas:
+        'made-for made-for made-for made-for'
+        'item1 item2 item3 item4';
+}
+
+#recents {
+    font-weight: bold;
+    font-family: sans-serif;
+    background: yellow;
+    height: 160px;
+    margin: 10px;
+
+    display: grid;
+    grid-template-columns: 25% 25% 25% 25%;
+    grid-template-rows: 33.3% 33.3% 33.3%;
```

Comment:
You could also do 1fr 1fr 1fr: that would make 3 rows that automatically divide up the available space evenly